### DISCUSSION TITLE
Why is `hub.docker.com` not updated?

### DISCUSSION THREAD

**Discussion by angeljqv** [soketi/pws#600] [Q&A] [answered]
https://hub.docker.com/r/soketi/pws/tags
Why is `hub.docker.com` not updated?

**Comment by rennokki** [comment, marked as answer]
`pWS` is the former name of Soketi, we wanted it to stay consistent. While we changed the name, we also migrated from Dockerhub because they started allowing organizations only on paid plans. quay.io seems a better solution for open source.

**Reply by angeljqv** [reply]
Thanks for the answer
